OpenRA.Game/Settings.cs

```diff
@@ -457,5 +458,51 @@ static void LoadSectionYaml(MiniYaml yaml, object section)
 
 			FieldLoader.Load(section, yaml);
 		}
+
+		internal void LoadModSettings(string mod, string settingsFile, Arguments args)
+		{
+			modSettingsFile = settingsFile;
+			ModSections = new Dictionary<string, object>()
```

Comment:
Hotkey settings are our most important mod-specific dependency.  Right now we expose hotkey configuration for all mods to all other mods, instead of letting each mod define the hotkeys that it actually uses.